src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.scss

```diff
@@ -0,0 +1,107 @@
+@import '../../../common/variables';
+@import './mixins';
+
+$avatar-spacing: 4px;
+
+.bcs-avatar-group {
+    display: flex;
+    margin: 10px 0;
+
+    .bcs-avatar-group-avatar {
+        height: 32px;
+        width: 32px;
+    }
+
+    .bcs-avatar-group-overflow-count {
+        align-items: center;
+        background: $bdl-neutral-02;
+        border-radius: 100%;
+        color: #fff;
+        cursor: pointer;
+        display: flex;
+        font-weight: bold;
+        justify-content: center;
+    }
+
+    .bcs-avatar-group-avatar-container {
+        margin-left: $avatar-spacing;
+
+        &:first-of-type {
```

Comment:
This pseudo-class works on elements tags rather than classes, so this is a bit misleading We can use sibling selectors instead to avoid the override entirely.

```
.bcs-avatar-group-avatar-container {
    & + & {
        margin-right: $avatar-spacing;
    }
}
```

Reply:
Done